lineapy/graph_reader/program_slice.py

```diff
@@ -262,27 +269,78 @@ def get_source_code_from_graph(program: Graph) -> CodeSlice:
                 )
             )
 
+    # print(slice_nodes)
```

Comment:
I think we should delete these type of code comments

Reply:
done in next cleanup commit